Add pub.dev filter to wasm page (#5919)
dart-lang/site-www: src/content/web/wasm.md

   contain a temporary workaround. For details, see
   [webdev issue 2206]({{site.repo.dart.org}}/webdev/issues/2296).

### Supported packages

> You can find packages that are compatible with Wasm by using the [`wasm-ready`][]
> filter on [pub.dev][].

Reviewer: **issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
To find Wasm-compatible packages,
use the [`wasm-ready`][] filter on [pub.dev][].
```
